<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QMainWindow" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>1003</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/assets/images/appicons/64x64.png</normaloff>:/assets/images/appicons/64x64.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_general">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1199</width>
             <height>952</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBox_enterKeySendsChat">
              <property name="text">
               <string>Pressing 'Enter' sends chat mesage</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enableNotifications">
              <property name="text">
               <string>Incoming message notifications</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enableAutoCloseChatWindows">
              <property name="text">
               <string>Auto-close chat window(s) on 15min of inactivity </string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_display">
       <attribute name="title">
        <string>Display</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1181</width>
             <height>934</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_textScaling">
                <property name="text">
                 <string>Text scaling</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_textScalingValue">
                <property name="text">
                 <string>1x</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="sliderTextScaling">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>4</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickInterval">
               <number>25</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enableInheritSystemTheme">
              <property name="text">
               <string>Inherit system theme</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enableDisplayAvatars">
              <property name="text">
               <string>Avatars</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enableSlim">
              <property name="text">
               <string>Slim mode (requires application restart)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enableGPUAccel">
              <property name="text">
               <string>GPU acceleration (requires application restart)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enableDisplayChatGradient">
              <property name="text">
               <string>Chat gradient (requires GPU acceleration)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Attachment preview</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enablePreview">
              <property name="text">
               <string>Enable preview</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enablePreviewUserInteraction">
              <property name="text">
               <string>Preview requires user-interaction</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enablePreviewInlineImages">
              <property name="text">
               <string>inline preview images</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>in-development:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_enableDisplayGroupchatJoinLeave">
              <property name="text">
               <string>Groupchat join/leave</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>630</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_theme">
       <attribute name="title">
        <string>Themes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_whatsthat_img">
              <property name="text">
               <string>whatsthat_img</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_theme_whatsthat">
              <property name="text">
               <string>WhatsThat</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">themeRadioGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_chatty_img">
              <property name="text">
               <string>chatty_img</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_theme_chatty">
              <property name="text">
               <string>Chatty</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">themeRadioGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_irssi_img">
              <property name="text">
               <string>irssi_img</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_theme_irssi">
              <property name="text">
               <string>Irssi</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">themeRadioGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Logging</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QScrollArea" name="scrollArea_logging">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1181</width>
             <height>934</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>The following options require a restart.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_logWriteToDisk">
              <property name="text">
               <string>Write to disk (~/.config/conversations/)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_logWriteToSyslog">
              <property name="text">
               <string>Emit to syslog</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_logTP">
              <property name="text">
               <string>Telepathy debug</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_logGlib">
              <property name="text">
               <string>GLib debug</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Note: For additional logging, you may compile conversations with the -DCMAKE_BUILD_TYPE=Debug option</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_quitConversations">
              <property name="text">
               <string>Quit conversations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_about">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_version">
          <property name="text">
           <string>Conversations version: -</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scroll">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="widget_changelog">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1183</width>
             <height>904</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="themeRadioGroup"/>
 </buttongroups>
</ui>
